ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+### Prerequisites
+
+* [ ] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+repo%3Aatom%2Felectron) to see if it is already reported?
```

Comment:
I think we can assume people have already searched existing issues before posting a new one, if they haven't, I doubt they would just ignore this.